bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -614,6 +620,73 @@ public ServerConfiguration setJournalFormatVersionToWrite(int version) {
         return this;
     }
 
+    /**
+     * Get max number of adds in progress. 0 == unlimited.
+     *
+     * @return Max number of adds in progress.
+     */
+    public int getMaxAddsInProgress() {
```

Comment:
This sounds like a stat rather than config. Maybe change getMaxAddsInProgressLimit

Reply:
will do